### PR TITLE
Protect: Resolve PHP notice for unknown var

### PR DESCRIPTION
**Pull request by kraftbj**
Fixes "Undefined variable: output in plugins/jetpack/modules/protect.php:411"

Pretty straight forward. If none of the headers are present, $output is never set. We need to set it to ensure it is there to return.

#### Changes proposed in this Pull Request:

*

#### Jetpack product discussion

no

#### Does this pull request change what data or activity we track or use?

no

#### Testing instructions:

* Load JP in a setup without any of those headers, probably like PHP CLI or soemthing.
*

### CODE REVIEW

**Comment by kraftbj** [comment]
I added it to 9.7 since I first saw it after installing the beta on w.org, but it's minor enough that I defer to the release team if it is worth the cherry-pick or not.

**Review by samiff — APPROVED** [review]
Looks good to me!

**Comment by samiff** [comment]
cherry-picked to `jetpack/branch-9.7`: e3d166aea22d3f39c14886e4674c4130573f452f

**Pull request merged**
